spec/models/movie_api_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
```

Comment:
I'm not sure what the convention is, but have seen something like this before:

`describe MovieApiService do
  context "instance methods" do
    describe ".conn" do
      it "returns 200 status" do
        api_response = MovieApiService.conn.get("/3/movie/top_rated")
        expect(api_response.status).to eq(200)
      end
   end
end`